packages/superset-ui-chart-controls/package.json

```diff
@@ -26,6 +26,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@emotion/react": "^11.4.1",
```

Comment:
We should import this as a peer dependency. We should also probably either use `^11.1.5` or consider making this version upgrade everywhere.

I think we also looked at just exporting Global from `@superset-ui/core` but I think there was something that got in the way of doing that last time we tried.

Reply:
Hey @rusackas. Thanks for your input. I do agree that we should export it from the core. It makes sense. For the sake of this PR, I have just made your suggested changes. Please have a look